### PR TITLE
feat: convert uri to uris

### PR DESCRIPTION
**Pull request by juzhiyuan**
related issue https://github.com/apache/apisix-dashboard/issues/717

related comments 

- https://github.com/apache/apisix-dashboard/issues/717#issuecomment-723812310
- https://github.com/apache/apisix-dashboard/issues/717#issuecomment-723938292

### CODE REVIEW

File: web/src/pages/Route/List.tsx [hunk 1]
```diff
@@ -35,24 +35,28 @@ const Page: React.FC = () => {
...
     {
       title: formatMessage({ id: 'page.route.domainName' }),
-      dataIndex: 'hosts',
       hideInSearch: true,
-      render: (_, record) =>
-        (record.hosts || []).map((host) => (
-          <Tag key={host} color="geekblue">
-            {host}
+      render: (_, record) => {
+        const list = record.hosts || (record.host && [record.host]) || [];
```
Comment (LiteSun) [inline web/src/pages/Route/List.tsx:40]: just want to confirm: Is it possible to return both hosts and hosts with different values?

File: web/src/pages/Route/List.tsx [hunk 2]
```diff
@@ -35,24 +35,28 @@ const Page: React.FC = () => {
...
     },
     {
       title: formatMessage({ id: 'page.route.path' }),
-      dataIndex: 'uri',
-      render: (_, record) =>
-        record.uris?.map((uri) => (
-          <Tag key={uri} color="geekblue">
-            {uri}
+      render: (_, record) => {
+        const list = record.uris || (record.uri && [record.uri]) || [];
```
Comment (LiteSun) [inline web/src/pages/Route/List.tsx:52]: Same question.

File: web/src/pages/Route/List.tsx (hunk 1 above)
Comment (juzhiyuan) [reply web/src/pages/Route/List.tsx:40]: If hosts exist, then use it, or use host.

**Review by liuxiran — APPROVED** [review]
LGTM

**Review by LiteSun — APPROVED** [review]

**Pull request merged**